src/js/rx/components/ConfirmRefillModal.jsx

```diff
@@ -13,11 +14,11 @@ class ConfirmRefillModal extends React.Component {
     // and possibly a server request.
     event.preventDefault();
     const alertContent = (
-      <span>
-        Refill for {this.props.drug} has been requested.
-      </span>
+      <b>
```

Comment:
The point wasn't to make the text bold, it's to use a shorter tag. It's all being re-styled in CSS. `b` has about the same semantic meaning as `span`.